VERSION

```diff
@@ -1 +1 @@
-version='0.8.14'
+version='0.8.15'
```

Comment:
@electricsam , please undo this version bump. The version now is dynamically updated, so we don't need to change it manually anymore. Thanks!